event-schema/event_schema.go

```diff
@@ -618,6 +636,27 @@ func getPrivateDataJSON(schemaHash string) []byte {
 }
 
 func (manager *EventSchemaManagerT) flushEventSchemas() {
+	// This will run forever. If you want to quit in between, change it to ticker and call stop()
+	// Otherwise the ticker won't be GC'ed
+	ticker := time.Tick(flushInterval)
+	for range ticker {
+
+		// TODO: Why we are throwing panics from the statements beneath us ?
+		// TODO: Refactored to use only a single dbHandle rather than creating the handle everytime. Ask Akash of the use case in first place ?
+		if !areEventSchemasPopulated {
```

Comment:
The access (both reads and write) to this variable seem to happen from multiple routines, it doesn't seem "data race" safe.